wandb/sdk/internal/internal_api.py

```diff
@@ -956,7 +956,10 @@ def push_to_run_queue(self, queue_name, launch_spec):
             for q in queues_found
             if q["name"] == queue_name
             # ensure user has access to queue
-            and (q["access"] == "PROJECT" or q["createdBy"] == self.default_entity)
+            and (
+                q["access"] in ["PROJECT", "USER"]
```

Comment:
Why this change? This will give access to all queues, since queue access is either PROJECT or USER

Reply:
If I don't set this I get "queue not found" errors with a launch queue I create through the UI. Guessing that shouldn't be the case?

Reply:
sounds like this should be reverted then?